russian/vesti-nedeli/vesti.md

```diff
@@ -0,0 +1 @@
+Вести вселяют смысл. Зефирная комната, подмышечный оркестр, ящик водки - грани единого плана, несмотря на неясные очертания,План несомненно велик. Разливаясь слегка тошнотворным теплом по гортани, водка вливает расслабляющую уверенность в каждую клеточку. Очертания Плана ярче не становятся, но грани вырастают до космических масштабов, разрывая границы понимания, уходя в мглу непознаваемого. Оркестр играл советскиЙ марш потом "Землю в иллюминаторе", гимн советской космонавтики незаметно влился в "Боже царя храни". Глаза закрылись. Все хорошо. Спи. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.